testcode/lib/pce/Makefile

```diff
@@ -0,0 +1,12 @@
+
+all: conio.pce
+
+conio.pce: conio.c
+	../../../bin/cl65 -t pce conio.c ../../../joy/pce-stdjoy.joy --mapfile conio.map -o conio.pce
```

Comment:
The static drivers are in the libraries.  We don't need to link to the dynamic ones (remove that
 `../../../joy/pce-stdjoy.joy`).